extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexus.spec.js

```diff
@@ -1,50 +1,56 @@
 describe('ext.wikia.adEngine.lookup.adapter.appnexus', function () {
 	'use strict';
 
-	function getAppNexus() {
-		return modules['ext.wikia.adEngine.lookup.adapter.appnexus'](
-			mocks.geo,
-			mocks.instantGlobals
-		);
-	}
-
 	var mocks = {
 		instantGlobals: {
 			wgAdDriverAppNexusBidderCountries: ['PL']
 		},
 		geo: {
 			isProperGeo: jasmine.createSpy('isProperGeo')
+		},
+		appNexusPlacements: {
+			getPlacement: function () {
+				return '123';
```

Comment:
123 it's a great value for mocks :D but when I saw it, I didn't know about that, so maybe add comment "mock data" or something like this.